go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/gravitational/shared-workflows
```

Comment:
I don't think this is correct still. We should have a go.mod per tool I feel, not a single one for the entire repo or one for each package.

Reply:
Oh I think I have a better idea of what is wanted here then. I'm thinking then it should be a module for libs as a whole and a module for each tool.

For example in this change two modules will be created.
```
github.com/gravitational/shared-workflows/libs
github.com/gravitational/shared-workflows/tools/changelog
```

Tools are versioned separately. Everything under `libs` will be versioned together.

Reply:
Btw I implemented the above. Check it out and let me know if it seems right.